packages/hs-react-ui/src/components/Dropdown/__tests__/__snapshots__/Dropdown.test.tsx.snap

```diff
@@ -1303,14 +1303,16 @@ exports[`Dropdown selects options from values prop 1`] = `
       <div
         class="sc-AxjAm sc-AxirZ c2"
       >
-        Bulbasaur, Charmander
+        Bulbasaur
+        , 
+        Charmander
```

Comment:
Is this going to be an issue, ie are we going to have to deal with weird line breakage? Just a sanity check.

Reply:
If we had white-space: prewrap or pre, it would look weird. Something to consider fixing though for sure.